orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/manifests/CreateBakeManifestTask.java

```diff
@@ -101,13 +105,9 @@ public TaskResult execute(@Nonnull Stage stage) {
 
     List<ExpectedArtifact> expectedArtifacts = context.getExpectedArtifacts();
 
-    if (expectedArtifacts == null || expectedArtifacts.isEmpty()) {
+    if (expectedArtifacts == null || expectedArtifacts.size() != 1) {
```

Comment:
This `null` check could be removed if you also defaulted `expectedArtifacts` in the stage context...but now I'm taking an awesome refactor and increasing the scope so feel free to ignore!

Reply:
:+1: :+1: done!

Reply:
(not done ignoring, done implementing your suggestion)